bench/README.md

```diff
@@ -22,36 +32,29 @@ Each write transactions is signed and thus requires:
 
 * an account with write permissions (e.g. Vendor account)
 * proper values for txn `sequence` which enforces txns ordering for an account
-
-By that reason load test uses prepared load data which can be generated as follows:
-
 * Initialize the pool and test accounts (**Warning** applicable to local in-docker pool only for now):
```

Comment:
Consider providing general instructions (not only for localnet) first and provide example with localnet

Reply:
Done! ✅